src/MockClassifier.UnitTests/InstitutionControllerTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using MockClassifier.Api.Controllers;
+using MockClassifier.Api.Models;
+using System.Linq;
+using Xunit;
+
+namespace MockClassifier.UnitTests
+{
+    public class InstitutionControllerTests
+    {
+        private readonly InstitutionController sut;
+
+        public InstitutionControllerTests()
+        {
+            sut = new InstitutionController();
+        }
+
+        [Theory]
+        [InlineData("")]
+        [InlineData("message1")]
+        [InlineData("message1","message2")]
+        public void ReturnsAccepted(params string[] messages)
+        {
+            // Arrange
```

Comment:
nice to see we use the same AAA pattern
I would prefer to have as few comments as possible though

Reply:
yeah, I would remove the comments because usually in my previous projects everyone follows AAA anyway and the segments are differentiated by a blank line. When you see that asserts block starts theres not much point of having a comment about it :D
This might be due to ecosystem differences though and if you guys are used to more comments then we can roll that way as well.

Reply:
I think given that we're only talking a few lines, I'd prefer to keep them to make it super clear. I do take your point, but this is OSS for everyone to see so no harm in being very specific.